### ISSUE TITLE
Slack?

### ISSUE THREAD

**Issue by trustin** [netty/netty#5493]
With [Slackin](https://github.com/rauchg/slackin) and Slack's IRC integration, it shouldn't be very hard to migrate from IRC.

**Comment by normanmaurer** [comment]
@trustin whats wrong with IRC ?

**Comment by trustin** [comment]
IRC isn't wrong. Slack is just better. Easier to:

- share images and files
- add pinned post
- use markdown in a message
- can have multiple channels (could have dedicated channel for github and teamcity notifications?)

It also has a nice official client and web application.

**Comment by trustin** [comment]
.. and a free plan allows us to archive up to 10000 messages.

**Comment by trustin** [comment]
Now I feel like I'm a salesman, dude. :-) (We can try Gitter as well, but people around me seem to have one or two Slack groups already, so..)

**Comment by vietj** [comment]
one issue with slack / IRC I found (when trying the integratio,) is that you don't have authenticatation on IRC via Slack (at least when we tried it).

So when user "foo" send a message on IRC from slack it appears as:

```
slackuser : <foo> hello everyone
```

on IRC.

**Comment by guidomedina** [comment]
Have you tried Gitter?
My main client uses Slack but if you want a better integration with Git then Gitter is OK, IMHO, not sure how "free" it is.

**Comment by trustin** [comment]
Yeah, I didn't like that each channel is tied to a Github repository last time I tried. Was there any changes since then?

**Comment by normanmaurer** [comment]
@trustin maybe Im just old-school but I still like irc the most ;) Works even from cmdline :)

**Comment by trustin** [comment]
:laughing: No worries. Let me open this for a few more days just in case @Scottmitch @nmittler or @buchgr wanna jump in.

**Comment by buchgr** [comment]
I have never used Slack or Gitter. So I can't comment on those. The only thing I can say is that I am not on IRC, because I tried a lot of clients and didn't like their user interface much. The number one thing that keeps me from using IRC though is the lack of a chat history. It's frustrating if I miss half of a discussion because my IRC client wasn't running. I think I would be more active on a chat system that supports chat history.

**Comment by guidomedina** [comment]
Try this, see how it looks for Akka guys to chat with Akka's users and contributors, and yes each channel is a project I think, but nothing like seeing it by yourselves, I'm quite sure many things have changed on Gitter:
- **Akka users:** https://gitter.im/akka/akka
- **Akka devs:** https://gitter.im/akka/dev

**Disclaimer:** I'm not selling Akka, I'm just trying to show an example of a Gitter user that uses it a lot.

**Comment by schmitch** [comment]
Yeah actually gitter isn't tied to a repository one can freely create repositories.
When you create a new chat room you have a choice of a Channel (Create a custom room, you can define permissions and add people individually.) or a Repository (Create a room from a GitHub repository.).

**Comment by sunng87** [comment]
Not even knew that Netty has an IRC channel. But if you want to change, I recommend Gitter. 
It has better integration with Github, you can get issues/prs referenced and highlighted. And there's mail notification for offline users to catch up.

**Comment by rkapsi** [comment]
+1 for Slack (and didn't know about Netty's IRC channel).

**Comment by AlmogBaku** [comment]
+1 Slack is a major improvement over IRC, and it looks like most of OSS projects moved to Slack.

**Comment by normanmaurer** [comment]
We have a discord now
